api/dedibox/v1/dedibox_utils.go

```diff
@@ -0,0 +1,99 @@
+package dedibox
+
+import (
+	"github.com/scaleway/scaleway-sdk-go/internal/async"
+	"github.com/scaleway/scaleway-sdk-go/internal/errors"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"time"
+)
+
+const (
+	defaultRetryInterval = time.Second * 15
+	defaultTimeout       = time.Minute * 30
+)
+
+type WaitForServiceRequest struct {
+	ServiceID     uint64
+	Zone          scw.Zone
+	Status        ServiceProvisioningStatus
```

Comment:
is this field necessary for this request?

Reply:
Fix :)